lisp_ease/src/queries/items.lisp

```diff
@@ -0,0 +1,71 @@
+; Defines all the items (objects) which have been used within the Virtual Reality and the axis object,
```

Comment:
Please use ``;;;`` for big block documentation.
``;;`` for one line documentation (or maybe 2-3).
``;`` for inline documentation.